fetcher/src/job/job.errors.ts

```diff
@@ -8,7 +8,6 @@ export class FetcherError extends Error {
 }
 
 export enum FetcherErrorCode {
-  IncompleteDataFeed,
```

Comment:
Is this update somehow related to the PoR?

Reply:
This is from fetcher. I realized that we did not used it anywhere tried to clean-up.

Reply:
Okay! Good good!

Reply:
it would be appreciated if you could make a separate PR for refactoring from next time :)